packages/eslint-plugin-clarity-adoption/src/rules/no-clr-button/disallowed-classes.ts

```diff
@@ -1,6 +1,21 @@
 export const primaryDisallowedClass = 'btn';
 
-export const additionalDisallowedClasses = ['btn-primary', 'btn-success', 'btn-warning', 'btn-danger'];
+export const additionalDisallowedClasses = [
```

Comment:
Why does this PR have changes to the eslint plugin together?

Reply:
No idea maybe is something for @sis0k0